The test nptl/tst-default-attr.c contained an unused value.  This is an
issue specific to glibc-rh990388-4.patch, which is not present upstream.

diff --git a/nptl/tst-default-attr.c b/nptl/tst-default-attr.c
index aeb52725f28a5f03..de63c638f48c03b2 100644
--- a/nptl/tst-default-attr.c
+++ b/nptl/tst-default-attr.c
@@ -110,8 +110,6 @@ run_threads (void)
 static int
 do_test (void)
 {
-  pthread_attr_t attr;
-
   pagesize = sysconf (_SC_PAGESIZE);
   if (pagesize < 0)
     {
